### ISSUE TITLE
Bug: chess.svg.board.arrows

### ISSUE THREAD

**Issue by PedanticHacker** [niklasf/python-chess#568]
I am reporting a bug that is not a logic bug, but rather a cosmetic bug. Let me explain.

I must give an _int_ value to my _self.square_ class instance attribute, which is then passed in twice to the _arrow_ argument of _chess.svg.board()_ as a tuple in a list, so that a circle is drawn in a square on the SVG chessboard. If I don't give _self.square_ an _int_ value, I get a _TypeError_. Well, the value of _self.square_ is changed later by the _chess.square()_ function [a mouse-click event handler takes care of that], but at initialization, I am forced to give an _int_ value and can not give a _None_ value to _self.square_.

Let me demonstrate with a boilerplate code.
```
class MyChessboard:
    def __init__(self):
        ...
        self.square = 0
        ...
        self.display_chessboard()
    def display_chessboard(self):
        ...
        self.board = chess.svg.board(..., arrows=[(self.square, self.square)], ...)
        ...
```
The result is that when my application starts and displays my SVG chessboard, there's always a circle drawn on the _a1_ square (because _0_ is the _a1_ square). If I give _self.square_ a value of _-20_, the circle is not seen on my SVG chessboard, but this is a very ugly approach. I'd be rather able to have `self.square = None` as its initial value.

So, in short, I'd like for the _arrows_ argument to accept _None_ as a passed-in value. Is that something we can do? That's a change that can occur before python-chess 1.0, right?

**Comment by niklasf** [comment]
Instead, do something like:
```python
arrows = [] if self.square is None else [(self.square, self.square)]
```
